lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -79,7 +79,6 @@ markdown==3.3.3; python_version >= "3.6"
 markupsafe==1.1.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0")
 mercurial==5.6.1
 mistune==0.8.4; python_version >= "3.6" and python_version < "4"
-mrcfile==1.2.0; python_version >= "3.6"
```

Comment:
That also need to stay here

Reply:
Ok, hopefully I have it correct now.  Thanks for the help!